modules/@apostrophecms/ui/ui/apos/scss/_normalize.css

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
This is also a package on `npm` (https://www.npmjs.com/package/normalize.css). Should we just import it from `node_modules`?

Reply:
importing from node_modules now